### ISSUE TITLE
Rendering of ImGui::ColorEdit3 changed

### ISSUE THREAD

**Issue by mklefrancois** [ocornut/imgui#5912]
Rendering of `ImGui::ColorEdit3` within a Dock, used to render like this:
![image](https://user-images.githubusercontent.com/38076163/203512764-e18fc632-606f-4d6a-bc31-7d1cb91980d6.png)

But after the change from 9b0c26b0b2adae3ccf66dc9552fae4945d735a0c,  (lines 5009, 5010 in `imgui_widget.cpp`)
```
        const float text_offset_x = (flags & ImGuiColorEditFlags_NoInputs) ? w_button : w_full + style.ItemInnerSpacing.x;
        window->DC.CursorPos = ImVec2(pos.x + text_offset_x, pos.y + style.FramePadding.y);
```
were changed for the following line
```
SameLine(0.0f, style.ItemInnerSpacing.x);
```

And the GUI item now renders as the following
![image](https://user-images.githubusercontent.com/38076163/203513090-f316d121-daaa-4c3e-91cc-b00562cfdf0c.png)

Maybe there is a misusage of the Docker on my side, but this is an unfortunate side effect. 

Is there something I can do to recover the original behavior? 

Kind regards
-mkl

**Comment by ocornut** [comment]
I confirm this is a regression broken by 54fb051e, only affect when `style.ColorButtonPosition == ImGuiDir_Left` which is not the default setting (feature added by e6a286b3).

Setting `style.ColorButtonPosition = ImGuiDir_Right;` as a temporary workaround with fix this.

**Comment by ocornut** [comment]
This is now fixed by 3a68574. Thank you for reporting!
